Assets/Scripts/CameraTransition.cs

```
@@ -0,0 +1,51 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using RunnerMovementSystem.Examples;
+
+public class CameraTransition : MonoBehaviour
+{
+    [SerializeField] private Transform _target;
+    [SerializeField] private float _timeToTransit;
+    [SerializeField] private float _rotationSpeed;
+    [SerializeField] private CameraFollowing _following;
+
+    private Player _player;
+
+    private void Start()
+    {
+        _player = FindObjectOfType<Player>();
```

Тут можно бы эксепшн кинуть если _player не найден. Fail Fast вместо ловли рандомных нуллрефов всегда хорошо.